docs/HowTo/Configure/Configure-TLS.md

```diff
@@ -0,0 +1,113 @@
+description: Configure TLS
+<!--- END of page meta data -->
+
+# Configure TLS
+
+Hyperledger Besu supports TLS for client and server communication. For example, you can
+[configure TLS](../../Concepts/TLS.md) for communication between EthSigner and Besu, and Besu and Orion.
+
+Configure TLS communication from the command line.
+
+**Prerequisites**:
+
+* Besu's password-protected PKSC #12 keystore.
+* File containing the keystore password
+
+## Configure Client TLS
+
+Allow clients (for example a dApp, curl, or EthSigner) to send and receive
+secure HTTP JSON-RPCs.
+
+**Prerequisites**:
+
+* The client must be configured for TLS.
+* Client's PKSC #12 keystore information.
+
+### Create the Known Clients File
+
+The known clients file allows clients with self-signed certificates or
+non-public certificates to connect to Besu.
+
+Create a file (in this example, `knownClients`) that lists one or more trusted
+clients. Use the format`<common_name> <hex-string>` where:
+
+* `<common_name>` is the Common Name specified in the client certificate.
+* `<hex-string>` is the SHA-256 fingerprint of the client certificate.
+
+!!! example
+    ```
+    ethsigner 8E:E0:85:9F:FC:2E:2F:21:31:46:0B:82:4C:A6:88:AB:30:34:9A:C6:EA:4F:04:31:ED:0F:69:A7:B5:C2:2F:A7
+    curl FC:18:BF:39:45:45:9A:15:46:76:A6:E7:C3:94:64:B8:34:84:A3:8E:B8:EA:67:DC:61:C0:29:E6:38:B8:B7:99
+    ```
+    
+You can use `openssl` or `keytool` to display the SHA256 fingerprint.
```

Comment:
Are there links we could include for openssl and keytool?

Reply:
Done